managing_multiple_clients.md

```diff
@@ -0,0 +1,342 @@
+# Managing Multiple MCP Clients
```

Comment:
While interesting - I feel like this should be taken into docs (which are coming) instead of a `.md` file in the root of the project